index.html

```diff
@@ -1,18 +1,90 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
+    <link rel="stylesheet" href="/styles/style.css">
+
     <!-- don't forget to link your styles -->
-  </head>
-  <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
-  </body>
-</html>
+</head>
+
+<body>
+    <div class="topnav">
+        <a href="#" class="logo"><img src="/images/spotify-logo.png"></a>
+        <div class="nav-right">
+            <a class="active" href="#home">Premium</a>
+            <a href="#Discover">Discover</a>
+            <a href="#Help">Help</a>
+            <a href="#Download">Download</a>
+        </div>
+    </div>
+
+
+    <div id="firstimage">
+        <div class="text">
+            <h1>Music for everyone.</h1>
+            <p class="desc">Spotify is now free on mobile, tablet and computer. Listen to the right music, wherever you are.</p>
+        </div>
+    </div>
+
+
+
+    <h2 class="top-title">What’s on Spotify?</h2>
+
+
+    <div class="firstSection">
+
+        <div class="box">
+            <img class="smallIcons" src="images/music-icon.png" alt="smallIcons" />
+            <h3 class="psecond">Millions of Songs</h3>
+            <p id="smallpar"> There are millions of songs on Spotify</p>
```

Comment:
It would be more correct to replace `id="smallpar"` with `class="smallpar"` because you're using it again on line 50 and 57. An id is supposed to be used only once